recipe/conda_build_config.yaml

```
@@ -487,6 +489,8 @@ libcrc32c:
   - 1.1
 libdap4:
   - 3.20.6
+libeantic:
+  - 1.0.3
```

Given the `run_exports` this should be simplified: https://github.com/conda-forge/e-antic-feedstock/blob/4729f28f3d73ced0a6bbb63929356337021cdc41/recipe/meta.yaml#L22
```suggestion
  - 1
```

---

Thanks, @xhochy, you are correct of course.